portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,45 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+/**
+* Creating a servlet that stores comments as a JSON list and use JavaScript that builds UI from
+* that data. 
+**/
```

Comment:
This comment should be before `public class DataServlet` rather than here.

Reply:
solved